nibabel/tests/test_scripts.py

```diff
@@ -68,7 +68,15 @@ def check_nib_ls_example4d(opts=[], hdrs_str=""):
 def test_nib_ls():
     yield check_nib_ls_example4d
     yield check_nib_ls_example4d, \
-          ['-H', 'dim,bitpix'], " \[  4 128  96  24   2   1   1   1\] 16"
+        ['-H', 'dim,bitpix'], " \[  4 128  96  24   2   1   1   1\] 16"
+    yield check_nib_ls_example4d, ['-c'], "", " 2:3 3:2 4:1 5:1.*"
```

Comment:
What is recommended way to run parametric tests nowadays in nibabel? Just call them?

On April 6, 2016 5:18:52 PM EDT, Matthew Brett <notifications@github.com> wrote:
>> @@ -68,7 +68,15 @@ def check_nib_ls_example4d(opts=[], hdrs_str=""):
>>  def test_nib_ls():
>>      yield check_nib_ls_example4d
>>      yield check_nib_ls_example4d, \
>> -          ['-H', 'dim,bitpix'], " \[  4 128  96  24   2   1   1  
>1\] 16"
>> +        ['-H', 'dim,bitpix'], " \[  4 128  96  24   2   1   1   1\]
>16"
>> +    yield check_nib_ls_example4d, ['-c'], "", " 2:3 3:2 4:1 5:1.*"
>
>Consider removing 'yield's here?  They make test failures much harder
>to debug
>
>---
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/nipy/nibabel/pull/437/files/0851f93c446db78ce7515bb5966ffa91dd043ba4#r58783774